types/create-hash/index.d.ts

```diff
@@ -1,16 +1,24 @@
 // Type definitions for create-hash 1.2
 // Project: https://github.com/crypto-browserify/createHash
 // Definitions by: BendingBender <https://github.com/BendingBender>,
-//                 Konstantin Yuriev <https://github.com/gallowsmaker>
-//                 Max Boguslavskiy <https://github.com/maxbogus>
+//                 Konstantin Yuriev <https://github.com/gallowsmaker>,
+//                 Max Boguslavskiy <https://github.com/maxbogus>,
+//                 Jordan Sexton <https://github.com/jordansexton>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.1
 
+/// <reference types="node" />
```

Comment:
Is it really required to use node types? This typings are used for browser version too.

Reply:
It's for `Buffer`, tsc throws otherwise. `Buffer` is a special case in the browser because Webpack usually automagically shims it. I noticed this is how the `secp256k1` package handles this issue, so I did the same: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/4b6756548680a242aaf29d7a8ee3d20b017a89ca/types/secp256k1/index.d.ts#L6

Reply:
I think that we should use https://github.com/feross/buffer for this case.
@VladimirGrenaderov what do you think?

Reply:
I don't see how this would declare that as a dependency, but also, `create-hash` does not do this and just relies on `Buffer` being available.

FWIW, this is also how it's done in several other packages, notably even https://github.com/feross/is-buffer

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/2206d001afbac3592a459dd68a99f742d46eff68/types/is-buffer/index.d.ts#L7

Reply:
(if I'm reading the code correctly?)

Reply:
Ah, it looks like internally it does this:

https://github.com/crypto-browserify/cipher-base/blob/bc18dc847050c34f11f7a0472ff13a76b53fedc6/index.js#L1

Reply:
However, looking at typings for `cipher-base` it still seems like this is the idiomatic way to do it:

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/4b6756548680a242aaf29d7a8ee3d20b017a89ca/types/cipher-base/index.d.ts#L6

Reply:
Ok, let's do this.

Reply:
I think you should push some small change to force Travis to rerun its build.

Reply:
I'll try that.

Reply:
Done!